SITCOM-1397_Extract_dominant_frequencies_from_sound_waves_in_video_associated_with_vibrations_on_Top-End_Assembly_II.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<p>As much as possible I suggest to avoid importing external libraies as librosa and moviepy. </p><p>If they are absolutely necessary I would be useful to state it clearly in a commentblock at the top of the notebook</p><p> </p><p>From what I have seen in the rest of the notebook, I suggest to consider using scipy.io.wavfile to deal with audio files and to use scipy.signal in place of librosa</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
The moviepy is bypassed in the new version of the notebook. The librosa python package has to be used, but it is commented at the top of the notebook.